tests/composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "require": {
-        "google/cloud": "^0.69.1"
+        "google/cloud": "^0.69.1",
```

Comment:
what do you think of installing `google/cloud` at `*` (i.e. latest) so that this test is essentially always integrating against the most recent version of Google Cloud PHP?

Reply:
Done. You are right. Thanks!
I thought `^` will use the latest version. But it turns out `^` has [limitations](https://getcomposer.org/doc/articles/versions.md#caret-version-range-). The test doesn't actually depend on the `google/cloud`. I am using the generated code `SpannerClient` so I don't need to generate them by myself each time.
I am going to clean all tests and add a `phpunit.xml.dist`/`bootstrap` later.